src/web/src/components/Posts/Timeline.tsx

```diff
@@ -52,7 +52,12 @@ const Timeline = ({ pages, totalPosts, nextPage }: Props) => {
   // elements to <Post>
   const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
     page.map(({ id, url }: Post) => (
-      <PostComponent postUrl={url} key={id} currentPost={++currentCount} totalPosts={totalPosts} />
+      <PostComponent
+        postUrl={url}
+        key={id}
+        currentPost={(currentCount += 1)}
```

Comment:
I think eslint is unhappy with this, where you are bundling an assignment into a return.  Probably we should make this side-effect free.  Can we get rid of `currentCount` somehow?  cc @DukeManh.

Reply:
I haven't thought of any other solution yet.

Reply:
Wrap your code with `{ }` brackets. See if that works

That eslint is to prevent functions that return a value from behaving unexpectedly.

Reply:
```suggestion
  const postsTimeline = pages.map((page): Array<ReactElement> | ReactElement =>
    page.map(({ id, url }) => {
      currentPost += 1;
      return (
        <PostComponent postUrl={url} key={id} currentPost={currentPost} totalPosts={totalPosts} />
      );
    })
  );
```